plasmapy/utils/datatype_factory_base.py

```diff
@@ -1,6 +1,6 @@
 # SunPy is released under a BSD-style open source licence:
 
-# Copyright (c) 2013-2018 The SunPy developers
+# Copyright (c) 2013-2019 The SunPy developers
```

Comment:
Well, technically SunPy has not yet extended the copyright dates in their license. Speaking of which, I found that and checked who changed it last year. I was as surprised as I would have been a year ago.